src/pages/iou/propTypes/index.js

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import CONST from '../../../CONST';
+import participantPropTypes from '../../../components/participantPropTypes';
+
+const iouPropTypes = PropTypes.shape({
+    /** ID (iouType + reportID) of the request */
+    id: PropTypes.string,
+
+    /** Amount of the request */
+    amount: PropTypes.number,
+
+    /** Currency of the request */
+    currency: PropTypes.string,
+
+    /** Description of the request */
+    comment: PropTypes.string,
+
+    merchant: PropTypes.string,
+    created: PropTypes.string,
+    receiptPath: PropTypes.string,
```

Comment:
This one should be "The path to an image of the receipt attached to the request"